scipy/sparse/_arrays.py

```diff
@@ -46,6 +46,10 @@ def _lil_container(self):
     def __mul__(self, *args, **kwargs):
         return self.multiply(*args, **kwargs)
 
+    def __rmul__(self, *args, **kwargs):
+        return self.multiply(*args, **kwargs)
+
+
```

Comment:
Too many blank lines (3).

Reply:
ah, that was my fault for merging the original fix without removing the extra line--I thought I could safely ignore the CI complaint because we do have some linting that only applies to new code/diffs, but apparently that was not the case for this one

Reply:
it does indeed make sense to include that fix in this PR if we're going to proceed here

Reply:
I added that fix to this PR.